lib/wfsc/falco_ctrl_planned_EFC.m

```diff
@@ -25,6 +25,22 @@
     % Step 2: For this iteration in the schedule, replace the imaginary part of the regularization with the latest "optimal" regularization
     % Step 3: Compute the EFC command to use.
     
+    %% SFR
+    try
+        if mp.Itr > mp.ctrl.start_iteration
+            mp.ctrl.dmfacVec = mp.ctrl.dmfacVecOn;
+        elseif mp.Itr == 1
+            mp.ctrl.dmfacVecOn = mp.ctrl.dmfacVec;
+            mp.ctrl.dmfacVec = 0;
+        elseif any(mp.Itr >= mp.est.itr_reset) && any(mp.Itr < mp.est.itr_reset + mp.ctrl.start_iteration)  
+            mp.ctrl.dmfacVec = 0;
+        else
+            mp.ctrl.dmfacVec = 0;
+        end
+    catch
+        
+    end
```

Comment:
This is a bit of a hack, it lets me turn off the controller for the first n iterations.  The `mp.ctrl.dmfacVec` is not actually a vector, if you want we can change it to a vector and then instead of this block it would just call `mp.ctrl.dmfacVec(Itr)`.

Reply:
I think we can leave this for now, but please Add a comment where the SFR is that explains it is used for EKF maintenance only.